libjulia/backends/evm/EVMCodeTransform.cpp

```diff
@@ -51,10 +70,19 @@ void CodeTransform::operator()(VariableDeclaration const& _varDecl)
 		while (variablesLeft--)
 			m_assembly.appendConstant(u256(0));
 	}
-	for (auto const& variable: _varDecl.variables)
+	// for (auto const& variable: _varDecl.variables)
+	for (auto it = _varDecl.variables.rbegin(); it != _varDecl.variables.rend(); ++it)
```

Comment:
You can use `for (auto const& variable: _varDecl.variables | boost::adaptors::reversed)`